test/validate_v1_v2_equality.ts

```diff
@@ -211,10 +273,17 @@ describe('Printings v1/v2 equality', () => {
   function validate(v1Field: string, maybeV2Field?: string) {
     const v2Field = maybeV2Field == undefined ? v1Field : maybeV2Field;
     v1CardsByCode.forEach((c, code) => {
+      if (checkTdc(c, code)) {
+        return;
+      }
       expect(c[v1Field], `${v2Field} mismatch for ${code}`).to.equal(printingsById.get(code)[v2Field]);
     });
   }
 
+  function checkTdc(v1: any, code: string): boolean {
```

Comment:
you have this function defined twice, but with different signatures.  Which do you need?

Reply:
They're in separate blocks:
- `describe('Card Sets v1/v2', () => {`
- `describe('Printings v1/v2 equality', () => {`